ValidatorPanel.php

```diff
@@ -858,11 +868,16 @@ public function stopBuffering()
 	/**
 	 * Validate.
 	 */
-	public function validate()
+	public function validate(\Nette\Application\Application $application)
```

Comment:
is the Application being passed here? I don't see it in https://github.com/Kdyby/HtmlValidatorPanel/blob/51df775e068a0c378c2ee00a59093d10d5089e24/DI/ValidatorExtension.php#L39

Reply:
I see 👍